meetings/2024-06-19.md

```diff
@@ -0,0 +1,130 @@
+* eP: but how do app devs know which to choose?
+* MdJ: app devs just publish their apps and their data needs, the rest is out of scope for them
+* eP: but what about data discovery by the app
+* MdJ: just support both
+* eP: we should agree on one
+* MdJ: It is not just between SAI and type indexes. There are web nodes, IPFS, etc. as an app developer you may hope that there was a single way but thats not the reality
+* eP: PA?
+* PA: I don't have much help to give here. but some thoughts: this is not a WG. The CG charter doesn't require us to choose a single solution. For WGs it's a necessity to reach agreement. The goal of a CG is more incubation. but I feel your pain, I started to develop my own Solid apps, and it's difficult to choose what to do, what to support. There's this mantra of be strict in what you produce and accepting in what you accept, but this diversity is a pain, we should try ot reach a balance between healthy incubation and the proverbial basar
+* MdJ: If you look at the possible outcomes, if we need to decide between type indexes or SAI. If we choose one the other group would be sad. I don't think non of it is an option. I couldn't choose either one.
+* eP: that's why I talk about BDD, we should work harder at finding consensus. It shouldn't be personal. We need a way to objectively evaluate proposals without getting attached ot any one.
+* Jesse: +1
+* PA: +1
+* PA: Solid has a broad scope. There may be different ways for multiple things to exist. Some use cases may be too niche, and others more desirable. that's also how consensus works
+* eP: maybe we need to introduce different storage types. but at least there would be a good foundation to explain it. even if we don't achieve unification, we should at least have due diligence
+* Jeff: I agree with PA it's a broad ecosystem and it's premature to pick one authorization mechanism. But I want to come back to the point about clients and servers. Suppose we want to give developers the choice between SAI and type indexes. How is that implemented? Some IDPs produce webid documents that don't link to type indexes, and other don't link to SAI. We don't have a clear pathway there. I'm in favour of persuing both approaches. There are plenty of places where there are multiple ways of doing things, I don't think that's a weakness
+* eP: I think the spec should require what is in the webid doc.
+* MdJ: yes, that makes sense.
+* Jeff: My opinion would be that if there is a proposal that has reached the level of SAI and the type indexes, a number of people have worked on it, in the CG and with implementations, then it's the responsibility for those writing webid docs to take that into account. Rather than deciding for one or the other, I think we should make it a strong recommendation to webid providers to include certain things in the webid doc, and allow clients to create their own third option. There has to be a way for a client to experiment with that.
```

Comment:
```suggestion
* eP: I think the spec should require what is in the WebID doc.
* MdJ: Yes, that makes sense.
* Jeff: My opinion is that, if there is a proposal that has reached the level of SAI and the Type Indexes, a number of people have worked on it, in the CG and with implementations, then it's the responsibility for those writing WebID docs to take that into account. Rather than deciding for one or the other, I think we should make it a strong recommendation to WebID providers to include certain things in the WebID document, and allow clients to create their own third option. There has to be a way for a client to experiment with that.
```